ui/app/models/totp-key.js

```diff
@@ -101,12 +134,15 @@ export default class TotpKeyModel extends Model {
   skew;
 
   @attr('boolean', {
+    editType: 'toggle',
     defaultValue: true,
+    subText: 'Vault will not return QR code and url upon key creation.',
   })
   exported;
 
   // Doesn't really make sense as we can generate our own QR code from the url
```

Comment:
We'll probably want to remove this comment unless we agree or know what it means 😂